src/features/filter.ts

```diff
@@ -1,10 +1,54 @@
-export const actions = { /* put action creators here */};
+import { Status } from '../types/Status';
 
-const filterReducer = () => {
-  return {
-    query: '',
-    status: 'all',
-  };
+enum ActionType {
+  ChangeQuery = 'filter/CHANGE_QUERY',
+  ChangeStatus = 'filter/CHANGE_STATUS',
+  DeleteQuery = 'filter/DELETE',
```

Comment:
you don't need this, when you want to delete query you can just change it to `''`